chia/_tests/util/full_sync.py

```diff
@@ -142,7 +142,8 @@ async def run_sync_test(
         config = load_config(root_path, "config.yaml")
 
         if test_constants:
-            constants = TEST_CONSTANTS
```

Comment:
I'm not sure this is a feasible path to go down. If you change how the test chains are generated, it may have ripple effects on a lot of tests. And you would need to update the `test-cache` as well.

Reply:
should i drop this, this was one of your changes